steps/cadence-innovus-cts/START.tcl

```diff
@@ -36,18 +36,8 @@ source innovus-foundation-flow/custom-scripts/setup-session.tcl
 # Execute
 #-------------------------------------------------------------------------
 
-# Read the execution order
-
-if {[ info exists env(order) ] && $::env(order) != "None" } {
-  set order $::env(order)
-} else {
-  # Default order of scripts
-  set order "
-    setup-ccopt.tcl
-    main.tcl
-    reporting.tcl
-  "
-}
+# Order is a comma-separated string containing scripts to run
+set order [split $::env(order) ","]
```

Comment:
Super minor, but can you go through and leave a newline between the comment and this line?

The coding convention here always separates comments from the following lines with a newline.